### PR TITLE
doc: added ffmpeg-qsv capability tables

### PR DESCRIPTION
**Pull request by dvrogozh**
Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>

### CODE REVIEW

File: doc/features/ffmpeg/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ffmpeg-qsv capabilities
+
+* [AV1](./av1.md)
+* [H264](./h264.md)
+* [H265](./h265.md)
```
Comment (dsocek) [inline doc/features/ffmpeg/README.md:5]: H.265 or better yet rename to H.265/HEVC

File: doc/features/ffmpeg/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# ffmpeg-qsv capabilities
```
Comment (dsocek) [inline doc/features/ffmpeg/README.md:1]: Maybe make the title of the landing page more official and to the point.  Something like:
"Codec capabilities of Intel® Quick Sync Video (QSV) with FFmpeg"

File: doc/features/ffmpeg/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+* [AV1](./av1.md)
+* [H264](./h264.md)
```
[the remark this answers is not in the document]
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done

File: doc/features/ffmpeg/README.md (hunk 1 above)
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done

File: doc/features/ffmpeg/README.md (hunk 2 above)
Comment (dvrogozh) [reply doc/features/ffmpeg/README.md]: done, but w/ slightly different title

**Pull request merged**